src/vs/workbench/parts/debug/browser/debugActions.ts

```diff
@@ -240,6 +251,9 @@ export class RestartAction extends AbstractDebugAction {
 	}
 
 	protected isEnabled(state: State): boolean {
+		if (!this.debugService.getViewModel().focusedProcess) {
```

Comment:
super.isEnabled(state) returns true - I don't believe we want to return StartAction.isEnabled when legitimately restarting the debugger and instead only return StartAction.isEnabled when the debugger is not running.  Thoughts?

Reply:
After some more thought, it should be okay for RestartAction.isEnabled to always return true if StartAction.isEnabled is true because RestartAction.run() will always be a valid call in that case.  As long as the proper run() function is called then it shouldn't matter if RestartAction.isEnabled returns true.